examples/python/device_selection.py

```diff
@@ -25,8 +25,8 @@ def create_default_device():
     Create default SyclDevice using `sycl::default_selector`.
 
     Device created can be influenced by environment variable
-    SYCL_DEVICE_FILTER, which determines SYCL devices seen by the
-    SYCL runtime.
+    ONEAPI_DEVICE_SELECTOR, which determines SYCL devices seen by the
```

Comment:
`SYCL_DEVICE_FILTER` still mentioned in `docs/_legacy/docfiles/user_guides/manual/dpctl/device_selection.rst` and `docs/_legacy/docfiles/user_guides/manual/dpctl/devices.rst` files.
Is it expected?

Reply:
`docs/_legacy` contains sources of `dpctl` from before gh-1619. This folder will be removed in near future.